metagpt/tools/web_browser_engine.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+
+from __future__ import annotations
+import asyncio
+import importlib
+
+from typing import Any, Callable, Coroutine, overload
+
+from metagpt.config import Config
+from metagpt.tools import WebBrowserEngineType
+from bs4 import BeautifulSoup
+
+
+class WebBrowserEngine:
+    def __init__(
+        self,
+        engine: WebBrowserEngineType | None = None,
+        run_func: Callable[..., Coroutine[Any, Any, str | list[str]]] | None = None,
+    ):
+        self.config = Config()
+        engine = engine or self.config.web_browser_engine
+
+        if engine == WebBrowserEngineType.PLAYWRIGHT:
+            web_browser_engine = importlib.import_module("metagpt.tools.web_browser_engine_playwright")
```

Comment:
the value name ends with `_engine` seems not well, actually it's a module.

Reply:
[remove inappropriate variable names used in the definition of the class WebBrowserEngine](https://github.com/geekan/MetaGPT/pull/69/commits/a6f13b693847298123f3b84deee571b8b17d880f)